### PR TITLE
20171214 bugfixes for master

### PR DESCRIPTION
**Pull request by daren-thomas**
Candidate for a release. Please check all scripts. Sensitivity Analysis doesn't work as per issue #930.

@JIMENOFONSECA Could you please have a look as I need a stable version to sent to veolia preferably tomorrow.

### CODE REVIEW

File: cea/default.config [hunk 1]
```diff
@@ -150,47 +152,58 @@ year.help = The year to base the calculations on. WARNING!, make sure you are us
...
 format-output.help = format of output files. csv is the default, HDF5 is used when data needs to be compressed.
+format-output.category = Advanced
 
 use-dynamic-infiltration-calculation = false
 use-dynamic-infiltration-calculation.type = BooleanParameter
 use-dynamic-infiltration-calculation.help = Use the dynamic infiltration calculation (slower!)
+use-dynamic-infiltration-calculation.category = Advanced
 
-use-daysim-radiation = true
+use-daysim-radiation = false
```
Comment (JIMENOFONSECA) [inline cea/default.config]: this should be true

File: environment.yml [hunk 2]
```diff
@@ -23,11 +23,12 @@ dependencies:
...
 - freexl=1.0.2=vc9_2
 - functools32=3.2.3.2=py27_1
-- gdal
+- geos
+- gdal=2.1.3=py27_vc9_7
 - geopandas>=0.2.1
-- geos>=3.5.1
 - hdf4=4.2.12=vc9_0
 - hdf5>=1.8.18
+- h5py
```
Comment (JIMENOFONSECA) [inline environment.yml]: I am not sure if we need this. I am using pandas.to_hdf and pandas.read_hdf to use hdf5

File: cea/default.config (hunk 1 above)
Comment (daren-thomas) [reply cea/default.config]: you're right... i was setting this to false for veolia - i'll just have to also explain how to install daysim. no problem.

File: environment.yml (hunk 2 above)
Comment (daren-thomas) [reply environment.yml]: We don't need it specifically... it seems to be included in one of our dependencies. I'm removing it from the list.

**Comment by daren-thomas** [comment]
@JIMENOFONSECA i have made the changes you requested and also run the tests. I think we're good to go. Next step: Integrate the sensitivity tool stuff...

**Pull request merged**